core/api/views/countries.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import mixins, generics
+
+from core.api.serializers import CountrySerializer
+from core.models import Country
+
+
+class CountryListAPIView(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    API endpoint that allows countries to be viewed.
+    """
+
+    serializer_class = CountrySerializer
+    queryset = Country.objects
+
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        return queryset.order_by("name")
```

Comment:
you can add order_by("name") in queryset declaration (line 13) and delete this method 😄